packages/react-components/react-breadcrumb/src/components/Breadcrumb/useBreadcrumb.ts

```diff
@@ -17,18 +18,28 @@ export const useBreadcrumb_unstable = (props: BreadcrumbProps, ref: React.Ref<HT
     dividerType = 'chevron',
     iconPosition = 'before',
     size = 'medium',
+    list,
     ...rest
   } = props;
+
+  const focusAttributes = useArrowNavigationGroup({
+    circular: true,
+    axis: 'horizontal',
+    memorizeCurrent: true,
+  });
+
   return {
     components: {
       root: 'nav',
       list: 'ol',
     },
     root: getNativeElementProps('nav', {
       ref,
+      'aria-label': 'breadcrumb' || props['aria-label'],
```

Comment:
But in this case even though the default could be 'breadcrumb', the condition is inverted, so the 'breadcrumb' value will always win here.

Reply:
yeah, that was a mistake. But I haven't fixed it yet because we didn't decide what `aria-label` should be

Reply:
I fixed it. `aria-label` is done according to the design spec. If we decide to change it, I'll update it in the following PRs